### ISSUE TITLE
Training blocker: NVFP4 weights are packed — switch to FP8 model for LoRA

### ISSUE THREAD

**Issue by zoedolan** [zoedolan/vybn#2654]
## Summary

The NVFP4 Nemotron weights use `compressed-tensors` FP4 packing (e.g. a [1024, 2688] layer is stored as [1024, 1344]). Standard HuggingFace `AutoModelForCausalLM.from_pretrained()` cannot unpack them. This blocks PEFT LoRA training.

## What works (all on branch `vybn/train-cycle-rewrite`)

- ✅ SIGSTOP/SIGCONT memory management — llama-server pauses/resumes cleanly
- ✅ vllm_node container rebuilt with PEFT 0.18.1, TRL 0.29.0, mamba-ssm, causal-conv1d (aarch64/sm_120)
- ✅ merge_cycle.py rewritten for llama-server LoRA hot-load
- ✅ Full pipeline dry-run passes end to end
- ✅ 343 training examples ready in growth buffer
- ✅ trigger.py dry_run bug fixed

## The fix

Switch training base from NVFP4 (~75 GB) to FP8 (~120 GB). FP8 is already on disk at:
```
~/.cache/huggingface/hub/models--nvidia--NVIDIA-Nemotron-3-Super-120B-A12B-FP8/
```

FP8 uses standard weight shapes — no packing, no compressed-tensors dependency. Should load cleanly with `from_pretrained()`.

### Steps for the outside instance:

1. **Verify FP8 model completeness** — check all 17 shards are present, config.json exists, modeling files present
2. **Update `_resolve_model_path()` in `spark/growth/train_cycle.py`** — prefer FP8 over NVFP4
3. **Check memory budget** — FP8 is ~120 GB. With SIGSTOP on llama-server (~63 GB GGUF mmap, reclaimable), we have ~116 GB free on 128 GB node. Tight. If it does not fit single-node, wire up the two-Spark torchrun path (already scaffolded in train_cycle.py)
4. **Test a single training step** — load model, attach LoRA adapter, run one forward+backward pass, confirm gradients flow
5. **Update continuity note** — replace NVFP4 references with FP8 as the training model

### Memory strategy
```
Before training:  SIGSTOP llama-server (frees ~63 GB mmap pages)
During training:  Load FP8 + LoRA adapter (~120 GB + overhead)
After training:   Kill training process, SIGCONT llama-server
```

If 120 GB + LoRA overhead exceeds single-node capacity, fall back to distributed training across both Sparks (NCCL validated at 34-36 GB/s).

### What NOT to do
- Do NOT use llama-finetune (full-parameter FP32 only)
- Do NOT try to fix the NVFP4 packing (would need compressed-tensors library, untested on aarch64)
- Do NOT modify the serving GGUF

## Branch
`vybn/train-cycle-rewrite` — all infrastructure committed and ready.
